examples/shell/openiotsdk/CMakeLists.txt

```diff
@@ -59,6 +59,11 @@ include(chip)
 
 add_subdirectory(${OPEN_IOT_SDK_EXAMPLE_COMMON}/app ./app_build)
 
+target_compile_definitions(openiotsdk-app
+    PRIVATE
+        IOT_SDK_APP_SERIAL_BAUDRATE=115200
```

Comment:
We can probably lower it, while working on another piece of code for the SDK, I discovered it was difficult to not drop characters in input if baudrate > 9600.

Reply:
I don't see any issues with dropping characters in CI now. It seems that this baud rate is good enough for the shell example.

Reply:
Sorry, you have right on this point. The error occurred in CI last time. I changed it to the proposed value and now It works.